cohortextractor/emis_backend.py

```diff
@@ -1165,6 +1165,10 @@ def quote(value):
         return str(value)
 
     value = str(value)
+
+    if re.match(r"^\w+ \+ interval '\d+' (day|month|year)$", value):
+        return value
+
```

Comment:
I don't know enough of the context here to comment really. I'm a bit surprised this all works but I'll trust you that it does.

Do you need to support negative intervals?

Is the idea that this is temporary and we'll move to the existing `<date> + 10 days` syntax at some point?

Reply:
Was sure I'd written an explanation / note of apology somewhere, sorry.

Yes, the idea is this is temporary.  Supporting `<date> + 10 days` will presumably come for free (with the code to actually generate the SQL as a rounding error) once we've done the big refactor.

Reply:
:+1: Makes sense